### ISSUE TITLE
Release tarballs contain specific attributes on '.'

### ISSUE THREAD

**Issue by raggi** [influxdata/influxdb#5252]
You cannot unpack the release tarballs blindly without having side effects on your system, for example:

```
tar: .: Cannot utime: Operation not permitted
tar: .: Cannot change mode to rwx------: Operation not permitted
tar: Exiting with failure status due to previous errors
```

For now I'd recommend users do:
```
tar zxf influxdb_0.9.6.1_x86_64.tar.gz ./influxdb_0.9.6.1_x86_64/
```

But that shouldn't be necessary.

Thanks!

**Comment by rossmcdonald** [comment]
@raggi What command did you run to get the original error output? I've attempted to recreate the issue here (both Mac and Linux):

```
$ curl -sLO http://influxdb.s3.amazonaws.com/influxdb_0.9.6.1_x86_64.tar.gz
$ tar -xvf influxdb_0.9.6.1_x86_64.tar.gz
x ./
x ./influxdb_0.9.6.1_x86_64/
x ./influxdb_0.9.6.1_x86_64/usr/
x ./influxdb_0.9.6.1_x86_64/usr/bin/
x ./influxdb_0.9.6.1_x86_64/usr/bin/influx
x ./influxdb_0.9.6.1_x86_64/usr/bin/influx_stress
x ./influxdb_0.9.6.1_x86_64/usr/bin/influxd
x ./influxdb_0.9.6.1_x86_64/usr/bin/influx_inspect
x ./influxdb_0.9.6.1_x86_64/usr/lib/
x ./influxdb_0.9.6.1_x86_64/usr/lib/influxdb/
x ./influxdb_0.9.6.1_x86_64/usr/lib/influxdb/scripts/
x ./influxdb_0.9.6.1_x86_64/usr/lib/influxdb/scripts/influxdb.service
x ./influxdb_0.9.6.1_x86_64/usr/lib/influxdb/scripts/init.sh
x ./influxdb_0.9.6.1_x86_64/etc/
x ./influxdb_0.9.6.1_x86_64/etc/influxdb/
x ./influxdb_0.9.6.1_x86_64/etc/influxdb/influxdb.conf
x ./influxdb_0.9.6.1_x86_64/etc/logrotate.d/
x ./influxdb_0.9.6.1_x86_64/etc/logrotate.d/influxdb
```

But it automatically gets unpacked in the current directory under the folder `influxdb_0.9.6.1_x86_64`. Are you seeing something different?

**Comment by raggi** [comment]
https://gist.github.com/c50cdfa0b968dc65479e will replicate the issue

output:
```
~ % bash influx-example.sh
working in /tmp/tmp.6fQEWGoqFX
total 8
drwxrwxr-x 2 root  raggi 4096 Jan  4 14:41 .
drwx------ 3 raggi raggi 4096 Jan  4 14:41 ..
./
./influxdb_0.9.6.1_x86_64/
./influxdb_0.9.6.1_x86_64/usr/
./influxdb_0.9.6.1_x86_64/usr/bin/
./influxdb_0.9.6.1_x86_64/usr/bin/influx
./influxdb_0.9.6.1_x86_64/usr/bin/influx_stress
./influxdb_0.9.6.1_x86_64/usr/bin/influxd
./influxdb_0.9.6.1_x86_64/usr/bin/influx_inspect
./influxdb_0.9.6.1_x86_64/usr/lib/
./influxdb_0.9.6.1_x86_64/usr/lib/influxdb/
./influxdb_0.9.6.1_x86_64/usr/lib/influxdb/scripts/
./influxdb_0.9.6.1_x86_64/usr/lib/influxdb/scripts/influxdb.service
./influxdb_0.9.6.1_x86_64/usr/lib/influxdb/scripts/init.sh
./influxdb_0.9.6.1_x86_64/etc/
./influxdb_0.9.6.1_x86_64/etc/influxdb/
./influxdb_0.9.6.1_x86_64/etc/influxdb/influxdb.conf
./influxdb_0.9.6.1_x86_64/etc/logrotate.d/
./influxdb_0.9.6.1_x86_64/etc/logrotate.d/influxdb
tar: .: Cannot utime: Operation not permitted
tar: .: Cannot change mode to rwx------: Operation not permitted
tar: Exiting with failure status due to previous errors
total 13108
drwxrwxr-x 3 root  raggi     4096 Jan  4 14:41 .
drwx------ 3 raggi raggi     4096 Jan  4 14:41 ..
drwx------ 4 raggi raggi     4096 Dec 10 15:40 influxdb_0.9.6.1_x86_64
-rw-r--r-- 1 raggi raggi 13386496 Jan  4 14:41 influxdb_0.9.6.1_x86_64.tar.gz
```

The archive contains "./" and it should not. It should only contain the influxdb subdirectory.

The following output further demonstrates the construction and evidence of a correct archive:

```
/tmp/tmp.6fQEWGoqFX/work % tar ztf influxdb_0.9.6.1_x86_64.tar.gz
./
./influxdb_0.9.6.1_x86_64/
./influxdb_0.9.6.1_x86_64/usr/
./influxdb_0.9.6.1_x86_64/usr/bin/
./influxdb_0.9.6.1_x86_64/usr/bin/influx
./influxdb_0.9.6.1_x86_64/usr/bin/influx_stress
./influxdb_0.9.6.1_x86_64/usr/bin/influxd
./influxdb_0.9.6.1_x86_64/usr/bin/influx_inspect
./influxdb_0.9.6.1_x86_64/usr/lib/
./influxdb_0.9.6.1_x86_64/usr/lib/influxdb/
./influxdb_0.9.6.1_x86_64/usr/lib/influxdb/scripts/
./influxdb_0.9.6.1_x86_64/usr/lib/influxdb/scripts/influxdb.service
./influxdb_0.9.6.1_x86_64/usr/lib/influxdb/scripts/init.sh
./influxdb_0.9.6.1_x86_64/etc/
./influxdb_0.9.6.1_x86_64/etc/influxdb/
./influxdb_0.9.6.1_x86_64/etc/influxdb/influxdb.conf
./influxdb_0.9.6.1_x86_64/etc/logrotate.d/
./influxdb_0.9.6.1_x86_64/etc/logrotate.d/influxdb
/tmp/tmp.6fQEWGoqFX/work % tar zcf corrected-influxdb_0.9.6.1_x86_64.tar.gz ./influxdb_0.9.6.1_x86_64/
/tmp/tmp.6fQEWGoqFX/work % tar zft corrected-influxdb_0.9.6.1_x86_64.tar.gz
./influxdb_0.9.6.1_x86_64/
./influxdb_0.9.6.1_x86_64/etc/
./influxdb_0.9.6.1_x86_64/etc/influxdb/
./influxdb_0.9.6.1_x86_64/etc/influxdb/influxdb.conf
./influxdb_0.9.6.1_x86_64/etc/logrotate.d/
./influxdb_0.9.6.1_x86_64/etc/logrotate.d/influxdb
./influxdb_0.9.6.1_x86_64/usr/
./influxdb_0.9.6.1_x86_64/usr/lib/
./influxdb_0.9.6.1_x86_64/usr/lib/influxdb/
./influxdb_0.9.6.1_x86_64/usr/lib/influxdb/scripts/
./influxdb_0.9.6.1_x86_64/usr/lib/influxdb/scripts/init.sh
./influxdb_0.9.6.1_x86_64/usr/lib/influxdb/scripts/influxdb.service
./influxdb_0.9.6.1_x86_64/usr/bin/
./influxdb_0.9.6.1_x86_64/usr/bin/influxd
./influxdb_0.9.6.1_x86_64/usr/bin/influx
./influxdb_0.9.6.1_x86_64/usr/bin/influx_stress
./influxdb_0.9.6.1_x86_64/usr/bin/influx_inspect
```

HTH

**Comment by rossmcdonald** [comment]
@raggi Understood, thanks for the clarification.

**Comment by raggi** [comment]
very welcome, thanks for the patience :)

**Comment by rossmcdonald** [comment]
This has been fixed with #6111, and is now in the latest nightlies.
